tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -597,7 +597,13 @@ EOF
 	[ -x "${AGENT_DEST}" ] || die "${AGENT_DEST} is not installed in ${ROOTFS_DIR}"
 	OK "Agent installed"
 
-	[ "${AGENT_INIT}" == "yes" ] && setup_agent_init "${AGENT_DEST}" "${init}"
+	if [ "${AGENT_INIT}" == "yes" ]; then
+		setup_agent_init "${AGENT_DEST}" "${init}"
+	else
+		# Setup systemd service for kata-agent
+		mkdir -p "${ROOTFS_DIR}/etc/systemd/system/basic.target.wants"
+		ln -sf "/usr/lib/systemd/system/kata-containers.target" "${ROOTFS_DIR}/etc/systemd/system/basic.target.wants/kata-containers.target"
```

Comment:
Does this have to be "{ROOTFS_DIR}/usr/lib/systemd/system/kata-containers.target" ?

Reply:
This is the same way it's done for the dracut overlay:

https://github.com/kata-containers/kata-containers/blob/main/tools/osbuilder/rootfs-builder/rootfs.sh#L459

The agent is setting it here:

Reply:
Never mind, makes sense to me how it is currently. Should be "target" itself.